.github/workflows/ci.yml

```diff
@@ -54,11 +54,12 @@ jobs:
         php-version: ['7.2', '7.3', '7.4', '8.0', '8.1', '8.2']
         composer-options: ['--prefer-stable']
         dependency-versions: ['highest']
-        include:
-          # testing lowest PHP version with lowest dependencies
-          - php-version: '7.2.5'
-            dependency-versions: 'lowest'
-            composer-options: '--prefer-lowest'
+# Disable lowest for now
+#        include:
+#          # testing lowest PHP version with lowest dependencies
+#          - php-version: '7.2.5'
+#            dependency-versions: 'lowest'
+#            composer-options: '--prefer-lowest'
```

Comment:
I could not get this to work, and perhaps we should also not support this any longer? 

I don't mind supporting php 7.2 as long as it uses the latest versions it can (since sf 5.4 support it) but supporting it with the lowest versions possible of the supporting libraries is rather annoying.

Reply:
Have you tried adding it back again after dropping support for symfony 4? Otherwise you might remove dead code

Reply:
I re-added it in 0a1f3c9f2c54e3a8cfffd7133cce841f43b60ab8.

I had to bump the minimum-requirement of doctrine/doctrine-bundle from 2.0.0 to 2.0.8 for the root-project and for the sf5.4-demo-project I updated the minimum-requirement  of symfony/flex to 1.21.

To have phpunit run succesfully, I had to signal phpunit-bridge to use phpunit 7 for that specific job.

Good of you to catch it, since this way we can guarantee our minimum requirements actually work. :+1: